web/src/beta/features/Dashboard/ContentsContainer/Members/ListItem.tsx

```diff
@@ -19,10 +70,50 @@ const ListItem: FC<{ member: TeamMember }> = ({ member }) => {
       </TypographyWrapper>
       <TypographyWrapper>
         <Typography size="body">
-          {member.role.charAt(0).toUpperCase() +
-            member.role.slice(1).toLowerCase()}
+          {isMemberRoleENG(memerRoleTranslation[member.role])
+            ? member.role.charAt(0).toUpperCase() +
+              member.role.slice(1).toLowerCase()
+            : memerRoleTranslation[member.role]}
         </Typography>
       </TypographyWrapper>
+      <TypographyWrapper>
+        <PopupMenu
+          label={
+            <Button icon="dotsThreeVertical" iconButton appearance="simple" />
+          }
+          menu={[
+            {
+              icon: "arrowLeftRight",
+              id: "changeRole",
+              title: t("Change Role"),
+              disabled: member.role === "OWNER",
+              onClick: () => handleUpdateRole(member)
+            },
+            {
+              icon: "close",
+              id: "remove",
+              title: t("Remove"),
+              disabled: member.role === "OWNER",
+              onClick: () => handleRemoveMember(member.userId)
+            }
+          ]}
+        />
+      </TypographyWrapper>
+      {addMemberModalVisible && (
+        <AddMemberModal
+          workspace={currentWorkSpace}
+          visible
+          onClose={() => setAddMemberModalVisible(false)}
+        />
+      )}
+      {updateRoleModalVisible && updatingRoleMember && (
```

Comment:
What's `updatingRoleMember`? I guess you mixed two implementaion together here.
- If you want to make `UpdateRoleModal` part of `ListItem`, this `updatingRoleMember` and related props and state in `index` all become unnecessary and should be removed.
- If you want to make `UpdateRoleModal` shared, it should be in `index`, and the local state `updateRoleModalVisible` should be removed. -> Let's go this way.